### PR TITLE
#8 fix test

### LINKED ISSUE #8: Reparat test
# Nevoie

Dupa ce am redenumit componentele si am creeat altele, testele pica.
# Livrabile
- [x] creem teste pentru componentele noi
- [x] stergem testele care nu mai sunt relevante

### PR DESCRIPTION
**Pull request by vtemian**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,5 @@ npm start
 
 - `gulp` - lints the code
 - `npm run build` - bundles the code for deploying
+- `npm test` - run tests using PhantomJS
+- `npm test-test` - run tests using Google Chrome and enable js debugger
```
Comment (palcu) [inline README.md]: `npm test-debug`

File: karma-debug.config.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+    /**
+     * Configuration option to turn off verbose logging of webpack compilation.
+     */
+    webpackMiddleware: {
+      noInfo: true
+    },
+
+    /**
+     * Once the mocha test suite returns, we want to exit from the test runner as well.
```
Comment (palcu) [inline karma-debug.config.js]: This comment is invalid.

File: karma-debug.config.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    /**
+     * The actual tests are preprocessed by the karma-webpack plugin, so that
+     * their source can be properly transpiled.
+     */
+    preprocessors: {
+      './test/config.js': ['webpack']
+    },
+
+    /**
+     * We want to run the tests using the PhantomJS headless browser.
```
Comment (palcu) [inline karma-debug.config.js]: This comment is invalid.

**Comment by palcu** [comment]
Finished review.

File: README.md (hunk 1 above)
Comment (vtemian) [inline README.md]: `npm run test-debug`

Comment (palcu) [inline README.md]: :+1:

**Comment by vtemian** [comment]
ping

File: test/components/test_alumni.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+describe("Alumni Component", function() {
+  it("should render the correct title", function() {
+    let component = TestUtils.renderIntoDocument(
+      <Alumni />
+    );
+
+    let heading = TestUtils.findRenderedDOMComponentWithTag(component, "h1");
+
+    expect(heading.getDOMNode().textContent).to.equal('Alumni');
```
Comment (palcu) [inline test/components/test_alumni.jsx]: Let's use `findDOMNode`.
https://facebook.github.io/react/docs/component-api.html#getdomnode
https://facebook.github.io/react/docs/working-with-the-browser.html#refs-and-finddomnode

**Comment by palcu** [comment]
LGTM

**Pull request merged**